apps/wallet/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "sui-wallet",
     "private": true,
-    "version": "0.1.1",
+    "version": "0.2.0",
```

Comment:
Why the big version bump, is this part of the mnemonic update?

Reply:
yes, probably we should have changed the major but since we are not at 1 yet this should be ok